vars/defaultIntegPipeline.groovy

```diff
@@ -58,7 +67,11 @@ def call(Map config = [:]) {
                                 } else {
                                     sh 'sudo usermod -aG docker $USER'
                                     sh 'sudo newgrp docker'
-                                    sh "sudo ./awsE2ESolutionSetup.sh --source-context-file './$source_context_file_name' --migration-context-file './$migration_context_file_name' --source-context-id $source_context_id --migration-context-id $migration_context_id --stage ${stageId} --migrations-git-url ${gitUrl} --migrations-git-branch ${gitBranch}"
+                                    def baseCommand = "sudo ./awsE2ESolutionSetup.sh --source-context-file './$source_context_file_name' --migration-context-file './$migration_context_file_name' --source-context-id $source_context_id --migration-context-id $migration_context_id --stage ${params.STAGE} --migrations-git-url ${params.GIT_REPO_URL} --migrations-git-branch ${params.GIT_BRANCH}"
```

Comment:
Can we make this one parameter per line to be easier to see whats changed or not?

Reply:
Added in 👍